editor/src/clj/editor/gui.clj

```diff
@@ -1783,22 +1809,24 @@
 
   (display-order (into base-display-order [:manual-size :enabled :visible :text :line-break :font :material :color :alpha :inherit-alpha :text-leading :text-tracking :outline :outline-alpha :shadow :shadow-alpha :layer]))
 
-  (output font-data font/FontData (g/fnk [font-datas font]
-                                    (or (font-datas font)
-                                        (font-datas ""))))
+  (output font-data font/FontData (g/fnk [costly-gui-scene-info font]
+                                    (let [font-datas (:font-datas costly-gui-scene-info)]
+                                      (or (get font-datas font)
+                                          (get font-datas "")))))
 
   ;; Overloaded outputs
   (output node-msg g/Any :cached produce-text-node-msg)
   (output gpu-texture TextureLifecycle (g/fnk [font-data] (:texture font-data)))
   (output scene-renderable-user-data g/Any :cached
-          (g/fnk [manual-size font font-shaders material material-shaders pivot text-data color+alpha]
+          (g/fnk [costly-gui-scene-info manual-size font material material-shader pivot text-data color+alpha]
```

Comment:
We can use the `material-shader` output from the base node here instead of looking it up ourselves.